src/en/glossary.md

```diff
@@ -2,17 +2,18 @@ Title: Glossary
 
 # Glossary of terms
```

Comment:
ok, let's go ahead and update it and I'll do some follow up work to remove the references in the author docs as a follow up PR